### DISCUSSION TITLE
Social Previewが適応されていた

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#143] [00: 未分類]
<img src="https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/140990100/2ec8c650-5d7a-4db4-98c3-5aa5d88520dd" width="400">

GitHubスマホアプリより

**Comment by kagomen** [comment]
おおおー！！良いですね👀✨

**Reply by kagomen** [reply]
ん？あれ？
#139 を今マージしたのですが、それより前にちゃんと反映されてたってことですか？

**Reply by pss-aileen** [reply]
すみません、こちらSocial Previewだけです...！何を思ったかOPGとタイトルにつけてました...！お騒がせしました！

**Reply by kagomen** [reply]
なるほど！
自分もSocial PreviewとOGPの違いよくわかってませんでした……💦

**Comment by kagomen** [comment]
すごい細かいんですが、Apple用？のアイコンは枠線なしの方がいいかも、ですかね❓
今はこのままでも良いかなーと思いますが！
![IMG_1041](https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/154225199/7e3465eb-8dd9-4dd8-9b02-9e6dca098291)

**Reply by pss-aileen** [reply]
ロゴが半透明どころかかなり透明で、黒色を背景にすると消えすぎてしまうなぁ...と思ったので背景で白をつけています。

あ、というか、枠線というのは、背景の白のことですか...？

**Reply by kagomen** [reply]
> 枠線というのは、背景の白のことですか...？

です、かね？
他のアプリのアイコンが枠いっぱいに表示されてるんですが、ファスコンのアイコンのみ枠内に枠？がある感じなのです

![IMG_1042](https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/154225199/d94c0aff-1bad-4725-b50a-ab98a6035900)

**Reply by kazzyfrog** [reply]
多分画像は用意されているけど、自分がサイズを間違えたかもです。。🙏

**Reply by kazzyfrog** [reply]
たくさん画像あったので、上記のzennやUdemyと同じようなサイズのアイコンもあるはず！！

**Reply by kagomen** [reply]
なんかバグかと思ったんですが、サイズの問題なら簡単そうですね！
忘れないようにイシューにしときます！

**Reply by kazzyfrog** [reply]
> 忘れないようにイシュー

助かります！！👍
